developer_manual/digging_deeper/continuous_integration.rst

```diff
@@ -11,6 +11,33 @@ We highly recommend setting up automated tests for your app, so that every chang
 * Unit testing: run unit tests for front-end and back-end where individual classes and components are tested in isolation
 * Integration testing: test components when they are combined
 
+Linting
+-------
+
+info.xml
+^^^^^^^^
+
+You can validate the ``info.xml`` :ref:`app metadata<app metadata>` file of an app with a simple github action:
+
+.. code-block:: yaml
+
+  name: Lint
+  on: pull_request
+
+  jobs:
+    xml-linters:
+      runs-on: ubuntu-latest
+      steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Download schema
+        run: wget https://apps.nextcloud.com/schema/apps/info.xsd
+      - name: Lint info.xml
+        uses: ChristophWurst/xmllint-action@v1
+        with:
+          xml-file: ./appinfo/info.xml
+          xml-schema-file: ./info.xsd
```

Comment:
I'm always confused by the yml indentation

```suggestion
        - name: Checkout
          uses: actions/checkout@master
        - name: Download schema
          run: wget https://apps.nextcloud.com/schema/apps/info.xsd
        - name: Lint info.xml
          uses: ChristophWurst/xmllint-action@v1
          with:
            xml-file: ./appinfo/info.xml
            xml-schema-file: ./info.xsd
```

Reply:
me too